mwdb/resources/user.py

```diff
@@ -160,6 +160,7 @@ def delete(self, login):
         """
         user_login_obj = load_schema({"login": login}, UserLoginSchemaBase())
 
+        notification = request.args.get('notification') == 'true'
```

Comment:
Don't do that manually. Make a Schema for `request.args` and use `field.Boolean()`. Marshmallow already have all the words that can be mapped to True.

Grep for `request.args` in other resources, some of them are parsed using Schema.